front/components/app/NewBlock.tsx

```diff
@@ -164,7 +170,7 @@ export default function NewBlock({
                     ))}
                   </div>
                 </div>
-                <div className="col-span-8 pr-2 text-sm text-gray-700 sm:col-span-9 sm:pl-3">
+                <div className="col-span-8 pr-2 text-sm text-gray-700 sm:col-span-9 sm:pl-6">
```

Comment:
This is because `database_schema` was too long?

Reply:
yup